tools/oas-apigee-mock/LICENSE.md

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
We don't need License files within the solutions. The DevRel repo already contains a root LICENSE file. Can we remove this?